### PR TITLE
requirements.txt - PICMI development version

### PR DESCRIPTION
**Pull request by ax3l**
Document in `requirements.txt` on how to install a pre-release version of PICMI.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,12 @@
+# basic dependencies
 numpy~=1.15
 periodictable~=1.5
+
+# PICMI
 picmistandard==0.0.16
+#   for development against an unreleased PICMI version, use:
+#picmistandard @ git+https://github.com/picmi-standard/picmi.git#subdirectory=PICMI_Python
```
Comment (ax3l) [inline requirements.txt:8]: Needs pip 20.1+
```
python3 -m pip install -U pip
```

Comment (dpgrote) [reply requirements.txt:8]: This is presumably a fix to use when building WarpX locally. The `development` branch should always depend on a specific version of picmi, correct?

Comment (ax3l) [reply requirements.txt:8]: Yes, correct.
This PR just adds a comment in `requirements.txt` how to change this for local builds, if one wants to.

The background is that we are extending PICMI for 1D simulations at the moment, and we realized we did not yet document that workflow for local testing of an experimental/upcoming PICMI version.

**Comment by ax3l** [comment]
@prkkumar wrote on Slack:
> I am getting this error now :
image (5)

**Comment by ax3l** [comment]
Install problem solved: we had an older PICMI 0.16.0 wheel still laying around locally.

Removed now, so the `... install *whl` does not pick it up.

**Review by prkkumar — APPROVED** [review]

**Pull request merged**